src/pages/messages/components/TicketDateIntervalFilter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box } from '@mui/material';
+import { DatePicker, DatePickerProps } from '@mui/x-date-pickers';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { formatDateStringToISO } from '../../../utils/date-helpers';
+
+const commonDatepickerProps: Partial<DatePickerProps<Date>> = {
+  format: 'dd.MM.yyyy',
+  views: ['year', 'month', 'day'],
+  disableHighlightToday: true,
+  slotProps: {
+    textField: { sx: { maxWidth: '10rem' }, size: 'small' },
+  },
+};
+
+export const TicketDateIntervalFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const maxDate = new Date();
+
+  const selectedDatesParam = searchParams.get(TicketSearchParam.CreatedDate);
+
+  const [startParam, endParam] = selectedDatesParam ? selectedDatesParam.split(',') : [];
+  const [selectedFromDate, setSelectedFromDate] = useState<string | undefined>(startParam ? startParam : '');
+  const [selectedToDate, setSelectedToDate] = useState<string | undefined>(endParam ? endParam : '');
+
+  const onChangeFromDate = (newDate: Date | null) => {
+    const newFromDate = newDate ? formatDateStringToISO(newDate) : undefined;
+    setSelectedFromDate(newFromDate);
+    updateSearchParams(`${newFromDate},${selectedToDate}`);
+  };
+
+  const onChangeToDate = (newDate: Date | null) => {
+    const newToDate = newDate ? formatDateStringToISO(newDate) : undefined;
+    setSelectedToDate(newToDate);
+    updateSearchParams(`${selectedFromDate},${newToDate}`);
+  };
+
+  const updateSearchParams = (newDateParam: string) => {
+    const [fromDate, toDate] = newDateParam.split(',');
+
```

Comment:
Nå lager `onChangeXxxxDate()` en ny streng med kommasepererte dates, og den brukes bare for å sende hit, hvor den splittes opp igjen, altså reversere det som ble gjort i forrige steg

```suggestion
  const updateSearchParams = (fromDate: string, toDate: string) => {
```